packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -447,8 +446,7 @@ function InstalledFonts() {
 							</Button>
 						) }
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> typogrpahy --> open font library modal and choose font

| Before | After     |
| :-------- | :------- |
| <img width="404" alt="installed-fonts-update-before" src="https://github.com/user-attachments/assets/8fe12179-c83f-4764-b695-ad0b4f77113f"> | <img width="404" alt="installed-fonts-update-after" src="https://github.com/user-attachments/assets/9cb78c85-f3a6-47a4-bf73-3e8313536236"> |